docs/release-process/CHECKLIST.md

```diff
@@ -0,0 +1,84 @@
+## Check list
+
+This section shows what things you need to do before creating a release. This is a manual process as CI does not test everything.
+
+The following example assumes we’re going from version 0.1.0 (`v0.1.0`) to
+0.2.0 (`v0.2.0`).
+
+- Checkout to old release tag
+  - e.g. `git checkout v0.1.0`
```

Comment:
All the commands are in this pattern like there is for e.g. and then a command. For the commands like above I understand user needs to make changes. But for other commands where we know for sure what the command is we should put it in a code block and not two back ticks.